### PR TITLE
Feature/board

### PR DESCRIPTION
**Pull request by eliciusc**

### CODE REVIEW

File: components/Board/Board.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+const Board = (props) => {
+    const { image, info } = props
+    return (
+        <div className={styles.container}>
+            <div>
+                <h3 className={styles.title}>{'Our Board'}</h3>
+            </div>
+            <div className={styles.board_box}>
+                <div className={styles.card_one}>
+                    <Line />
```
Comment (marceloglacial) [inline components/Board/Board.jsx:14]: You don't need a component for it. Please use CSS

File: components/Board/Board.jsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import styles from '@/styles/Board.module.css'
+import Card from '@/components/Board/Card'
+import Line from '@/components/Board/Line'
+
+const Board = (props) => {
+    const { image, info } = props
+    return (
+        <div className={styles.container}>
+            <div>
+                <h3 className={styles.title}>{'Our Board'}</h3>
```
Comment (marceloglacial) [inline components/Board/Board.jsx:10]: Please use props for the title

File: components/Board/Board.jsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+            </div>
+            <div className={styles.board_box}>
+                <div className={styles.card_one}>
+                    <Line />
+                    <Card image={image.marcelo} info={info.marcelo} />
```
Comment (marceloglacial) [inline components/Board/Board.jsx:15]: Please use an array of objects for the cards info

File: components/Board/Card.jsx [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    return (
+        <div className={styles.card_box}>
+            <div className={styles.image_box}>
+                <Image src={image.src} alt={image.alt} width={image.width} height={image.height} />
+            </div>
+            <div className={styles.center_box}>
+                <div className={styles.title_box}>
+                    <h5 className={styles.title_text}>{info.title}</h5>
+                </div>
+                <div>
```
Comment (marceloglacial) [inline components/Board/Card.jsx:16]: If you are not identifying a div, you maybe not need it

File: components/Board/Line.jsx [hunk 5]
```diff
@@ -0,0 +1,4 @@
+import styles from '@/styles/Board.module.css'
+
+const ColoredLine = () => <div className={styles.line}></div>
```
Comment (marceloglacial) [inline components/Board/Line.jsx:3]: Please use CSS instead of a component

File: styles/Board.module.css [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+    flex-direction: column;
+    align-items: center;
+    justify-content: flex-start;
+    text-align: justify;
+    color: var(--button-bg-outline-disabled);
+    gap: calc(3.125 * var(--spacing-unit));
+}
+
+.image_box {
+    width: auto;
```
Comment (marceloglacial) [inline styles/Board.module.css:33]: Please use squared images and use CSS to get it rounded

**Review by marceloglacial — CHANGES_REQUESTED** [review]

File: components/Board/Board.jsx (hunk 1 above)
Comment (eliciusc) [reply components/Board/Board.jsx]: Removed

File: components/Board/Board.jsx (hunk 2 above)
Comment (eliciusc) [reply components/Board/Board.jsx]: Done

File: components/Board/Board.jsx (hunk 3 above)
Comment (eliciusc) [reply components/Board/Board.jsx]: Done. Thanks for the sandbox!

File: components/Board/Card.jsx (hunk 4 above)
Comment (eliciusc) [reply components/Board/Card.jsx:16]: image

That's odd. I have a "p" tag inside

Comment (eliciusc) [reply components/Board/Card.jsx:16]: Removed

File: styles/Board.module.css (hunk 6 above)
Comment (eliciusc) [reply styles/Board.module.css:33]: I didn't find the original files

File: components/Board/Board.jsx [hunk 7]
```diff
@@ -0,0 +1,17 @@
+import styles from '@/styles/Board.module.css'
+import Card from '@/components/Board/Card'
+
+const Board = (props) => {
+    const { cards, title } = props
+    return (
+        <div className={styles.container}>
+            <h3 className={styles.title}>{title}</h3>
+            <div className={styles.board_box}>
+                {cards.map((card) => (
```
Comment (marceloglacial) [inline components/Board/Board.jsx:10]: Please add error boundaries (i.e.: when there's no array). You can use a question mark on the map function)
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

File: components/Board/Card.jsx [hunk 8]
```diff
@@ -0,0 +1,34 @@
+import { Icon } from '@/components'
+import Image from 'next/image'
+import styles from '@/styles/Board.module.css'
+
+const Card = (props) => {
```
Comment (marceloglacial) [inline components/Board/Card.jsx:5]: Please move this file to a Card folder and add it to the index

**Review by marceloglacial — CHANGES_REQUESTED** [review]

File: components/Board/Board.jsx (hunk 7 above)
Comment (eliciusc) [reply components/Board/Board.jsx]: Done

File: components/Board/Card.jsx (hunk 8 above)
Comment (eliciusc) [reply components/Board/Card.jsx]: Done

**Comment by marceloglacial** [comment]
Please resolve the conflicts

File: styles/Board.module.css [hunk 9]
```diff
@@ -0,0 +1,119 @@
+.container {
```
Comment (marceloglacial) [inline styles/Board.module.css:1]: Please, fix the spacing:

Yours:
Screenshot 2023-03-13 at 11 04 18 PM

Prototype:
Screenshot 2023-03-13 at 11 04 53 PM

**Review by marceloglacial — CHANGES_REQUESTED** [review]

File: styles/Board.module.css (hunk 9 above)
Comment (eliciusc) [reply styles/Board.module.css:1]: Fixed

**Review by marceloglacial — APPROVED** [review]

**Pull request merged**